[develop] Add smoke and dust verification (#1174)
ufs-community/ufs-srweather-app: doc/UsersGuide/BuildingRunningTesting/RunSRW.rst

that attempt is successful, the workflow will move on to subsequent tasks.  Thus:

   * Users who have the obs files already available (staged) on their system only need to set ``*_OBS_DIR``
     and ``OBS_*_FN_TEMPLATES[1,3,...]`` in ``config.yaml`` to match those staging locations and file names.
   
   * Users who do not have the obs files available on their systems and do not have access to NOAA HPSS
     need to download :term:`CCPA`, :term:`NOHRSC`, :term:`MRMS`, :term:`NDAS`, :term:`AERONET`, and/or
     :term:`AIRNOW` files manually from collections of publicly available data.
     Then, as above, they must set ``*_OBS_DIR`` and ``OBS_*_FN_TEMPLATES[1,3,...]`` to match those
     staging locations and file names.

.. note::

>    AIRNOW observations can be retrieved from AWS in addition to HPSS, but this requires changing some default settings.
>    See ``ush/config_defaults.yaml`` or :numref:`Section %s <GeneralVXParams>` for more details. 

Reviewer: ```suggestion
   AIRNOW observations can be retrieved from AWS or HPSS, but retrieving from AWS requires changing some default settings.
   See ``ush/config_defaults.yaml`` or :numref:`Section %s <GeneralVXParams>` for more details. 
```
